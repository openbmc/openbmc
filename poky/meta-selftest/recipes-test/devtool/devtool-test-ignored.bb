LICENSE = "CLOSED"
INHIBIT_DEFAULT_DEPS = "1"

SRC_URI = "file://${BPN}.tar.gz \
           file://${BPN}.patch"

S = "${WORKDIR}/${BPN}"

EXCLUDE_FROM_WORLD = "1"
