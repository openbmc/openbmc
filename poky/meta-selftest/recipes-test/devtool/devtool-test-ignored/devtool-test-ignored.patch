diff --git a/ignored b/ignored
index a579759..e3d7b43 100644
--- a/ignored
+++ b/ignored
@@ -1 +1 @@
-I'm so ignored
+# I'm so ignored
