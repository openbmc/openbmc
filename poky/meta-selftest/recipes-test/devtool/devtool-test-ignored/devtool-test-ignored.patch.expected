From 3a286343cc5cadd83f41d524ee3606ae51df9ee7 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Thu, 28 May 2020 01:32:31 +0200
Subject: [PATCH] meta-selftest: add test of .gitignore in tarball

---
 ignored | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ignored b/ignored
index a579759..e3d7b43 100644
--- a/ignored
+++ b/ignored
@@ -1 +1 @@
-I'm so ignored
+# I'm so ignored
